src/Send/ConfirmScreen/ConfirmScreen.stories.tsx

```diff
@@ -13,14 +12,15 @@ storiesOf('ConfirmScreen', module).add('Default', () => {
     key: 'key',
     name: 'name',
     params: {
-      defaultAssetAmount: new BigNumber('1111111111'),
-      balanceAfterTx: new BigNumber('10'),
+      defaultAssetAmount: '1111111111',
+      balanceAfterTx: '10',
       address: 'address_123123123',
-      availableAmount: new BigNumber('1111111100'),
-      fee: new BigNumber('1'),
-      tokens: tokenEntries,
+      availableAmount: '1111111100',
+      fee: '1',
+      selectedTokens: balance,
```

Comment:
what is `selectedTokens`?
is this different than `balances`?

Reply:
https://github.com/Emurgo/yoroi-mobile/blob/d3f885a90087d66d306e0cb225818ce8cc4f093f/src/Send/SendScreen/SendScreen.tsx#L167

The tokens to be sent

Reply:
Renamed